analysis/study_definition.py

```diff
@@ -83,11 +87,83 @@
+    agegroup = patients.categorised_as(
+        {
+            "18-39": "age >= 18 AND age < 40",
+            "40-49": "age >= 40 AND age < 50",
+            "50-59": "age >= 50 AND age < 60",
+            "60-69": "age >= 60 AND age < 70",
+            "70-79": "age >= 70 AND age < 80",
+            "80plus": "age >= 80",
+            "missing": "DEFAULT",
+        },
+        return_expectations={
+            "rate": "universal",
+            "category": {
+                "ratios": {
+                    "18-39": 0.17,
+                    "40-49": 0.17,
+                    "50-59": 0.17,
+                    "60-69": 0.17,
+                    "70-79": 0.17,
+                    "80plus": 0.15,
+                    "missing": 0,
+                }
+            },
+        },
+    ),   
+    ### age group (used for age standardisation)
+    agegroup_std = patients.categorised_as(
+        {
+            "15-19 years": "age >= 15 AND age < 20", # (age is here always >= 18 by definition)
+            "20-24 years": "age >= 20 AND age < 25",
+            "25-29 years": "age >= 25 AND age < 30",
+            "30-34 years": "age >= 30 AND age < 35",
+            "35-39 years": "age >= 35 AND age < 40",
+            "40-44 years": "age >= 40 AND age < 45",
+            "45-49 years": "age >= 45 AND age < 50",
+            "50-54 years": "age >= 50 AND age < 55",
+            "55-59 years": "age >= 55 AND age < 60",
+            "60-64 years": "age >= 60 AND age < 65",
+            "65-69 years": "age >= 65 AND age < 70",
+            "70-74 years": "age >= 70 AND age < 75",
+            "75-79 years": "age >= 75 AND age < 80",
+            "80-84 years": "age >= 80 AND age < 85",
+            "85-89 years": "age >= 85 AND age < 90",
+            "90plus years": "age >= 90",
+            "missing": "DEFAULT",
+        },
+        return_expectations={
+            "rate": "universal",
+            "category": {
+                "ratios": {
+                    "15-19 years": 0.05,
+                    "20-24 years": 0.05,
+                    "25-29 years": 0.05,
+                    "30-34 years": 0.05,
+                    "35-39 years": 0.05,
+                    "40-44 years": 0.1,
+                    "45-49 years": 0.1,
+                    "50-54 years": 0.1,
+                    "55-59 years": 0.1,
+                    "60-64 years": 0.05,
+                    "65-69 years": 0.05,
+                    "70-74 years": 0.05,
+                    "75-79 years": 0.05,
+                    "80-84 years": 0.05,
+                    "85-89 years": 0.05,
+                    "90plus years": 0.05,
+                    "missing": 0,
+                }
+            },
+        },
+    ),
     ### sex 
     sex = patients.sex(
         return_expectations={
             "rate": "universal",
-            "category": {"ratios": {"M": 0.49, "F": 0.51}},
+            "category": {"ratios": {"Male": 0.49, "Female": 0.51}},
```

Comment:
Note that on dummy data this will return "Male" or "Female", but on real data, this variable will return "M" or "F"

Reply:
Oh, that's really good to know. Changed back to M/F.